src/Core/Extensible.php

```diff
@@ -135,6 +135,10 @@ protected function defineExtensionMethods()
                 $this->addCallbackMethod($method, function ($inst, $args) use ($method, $extensionClass) {
                     /** @var Extensible $inst */
                     $extension = $inst->getExtensionInstance($extensionClass);
+                    if (!$extension) {
```

Comment:
Under what circumstances would there not be an extension instance?

Reply:
You can reproduce it with the advancedworkflow comment above - the method can return null though, so this is a defensive patch